strength_percentiles.py

```diff
@@ -94,40 +106,82 @@ def get_data_from_table(row, column):
-            print "\t", result
-        print "\n"*5
-    print "Number of lifters: ", len(container['all'])
-    print "Number of men: ", len(container['men'])
-    print "Number of women: ", len(container['women'])
-    print "Number of amateurs: ", len(container['amateurs'])
-    print "Number of professionals: ", len(container['professionals'])
-    print "Number of raw lifters: ", len(container['raw'])
-    print "Number of equipped lifters: ", len(container['equipped'])
+    """
+    table_deletion_string = "drop table if exists meet_results"
+
+    create_table_string = """CREATE TABLE meet_results (
+    lifter_id INTEGER PRIMARY KEY, gender TEXT,
+    professional_status TEXT, equipment TEXT, squat REAL,
+    bench REAL, deadlift REAL, total REAL)
+    """
+
+    # IMPORTANT NOTE: Instead of using the SQLite's parameterized query
+    # feature, where the ?'s are placeholders for the parameters and the
+    # parameters are supplied in an iterable as the second argument to the
+    # cursor's execute method, I also had the option of formatting the command
+    # using Python string formatting, to insert the information I collected
+    # from online into the command string.  However, this would expose my
+    # computer to risks similar to those posed by Python 2's input function.
+    # If I happened to collect malicious SQL code from the internet, it would
+    # be formatted into the command and executed.  SQLite's parameterized
+    # query feature allows me to harmlessly insert the information I collect
+    # online into a database without executing it.
+    result_storage_string = """INSERT INTO meet_results (
+    gender,
+    professional_status,
+    equipment,
+    squat,
+    bench,
+    deadlift,
+    total)
+
+    Values (
+    ?,
+    ?,
+    ?,
+    ?,
+    ?,
+    ?,
+    ?
+    )
+    """
+    connection = sq.connect(database)
+    cursor = connection.cursor()
+    cursor.execute(table_deletion_string) # Delete table if it already exists
+                                    # if we attempt to create a table with a
+                                    # name that is already in use an error will
+                                    # result.  The remade table will be more
+                                    # up-to-date anyway.
+    cursor.execute(create_table_string)
+
+    page = urllib2.urlopen(webpage)
+    soup = BeautifulSoup(page, 'html.parser')
+    meet_results_table = soup.find_all('tr') # all rows of meet results table,
+                                             # each corresponds to an
+                                             # individual's performance at the
+                                             # meet
+    for row in meet_results_table: # information for one lifter
+        result_and_categories = parse_result_and_categories(row)
+        if result_and_categories: # row contains results of meet
+            cursor.execute(result_storage_string, result_and_categories)
```

Comment:
Here there's the implicit guarantee that the hard coded result storage string and the return from parse_result_and_categories take arguments in the same order. But this guarantee is just silently there. You can imagine this might be kind of annoying to double check.
I think using named placeholders would be more clear(and less prone to bugs).